.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8"]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
can close:
If any further complications show up with python 3.7, then I suggest to just remove python 3.7 from the supported versions

Reply:
I was not blocked by the 3.7 version except for the mock of `async with`. I was able to resolve all others problems. We will see in future if libraries will be not compatible with py3.7 but at instant it is, so i prefer keep it too.